sympy/physics/control/lti.py

```diff
@@ -1656,31 +1656,61 @@ def __neg__(self):
         return MIMOParallel(*arg_list)
 
 
-class Feedback(Basic):
-    """
-    A class for representing negative feedback interconnection between two
-    input/output systems. The first argument, ``num``, is called as the
-    primary plant or the numerator, and the second argument, ``den``, is
-    called as the feedback plant (which is often a feedback controller) or
-    the denominator. Both ``num`` and ``den`` can either be ``Series`` or
+class Feedback(SISOLinearTimeInvariant):
+    r"""
+    A class for representing closed-loop feedback interconnection between two
+    SISO input/output systems. The first argument, ``plant``, is the
```

Comment:
This is typically true however, in the block diagram algebra the first item doesn't have to be the plant. Nor the second should be controller. You could very well composing different parts of a dynamical system. Hence this might confuse users. sys1, sys2 terminology is fine I think. 

Besides in the SISO case, the order won't matter since closed loop is GK/(1+GK) and G and K commutes.

Reply:
I understand what you are trying to say. I initially chose the sys1-sys2 naming convention since it was used in both MATLAB and python-control. I went ahead with the `plant` and `feedback_controller` convention as the goal was to make it beginner-friendly. 
`plant` and `feedback_controller` made more sense than `plant` and `controller` because the controller doesn't need to be always in the feedback path. Some controller systems can also be placed in the feedforward path. `feedback_controller` specified this fact.

Reply:
If `plant`-`controller` is not a general terminology, then maybe we can go with the `open_loop_gain`-`feedback_gain` or `sys1`-`sys2`. Whichever suits better.

Reply:
I think we can go with sys1-sys2 terminology.

Reply:
G-H is also fine if we do a very simple ASCII-art of the loop

Reply:
> G-H is also fine if we do a very simple ASCII-art of the loop

Slightly off-topic but always wanted to know whether there is some tool for creating block diagram using ASCII art.